src/js/reducers/app-state-reducer.js

```diff
@@ -0,0 +1,80 @@
+import { ENABLE_SETTINGS, DISABLE_SETTINGS, TOGGLE_WIDGET, OPEN_WIDGET, CLOSE_WIDGET, SHOW_SETTINGS, HIDE_SETTINGS, SHOW_SETTINGS_NOTIFICATION, HIDE_SETTINGS_NOTIFICATION, SET_SERVER_URL, SET_EMAIL_READONLY, UNSET_EMAIL_READONLY, UPDATE_READ_TIMESTAMP } from 'actions/app-state-actions';
```

Comment:
why do action consts need to be in separate files?
what if we had a single `actions.js` one to rule them all
then you could `import * as action from 'actions'`